inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java

```diff
@@ -36,7 +36,7 @@
 public class Trie<V>
```

Comment:
This class has still a method that starts with an underscore (`_getNode()`). I'd suggest to change that too.